src/native/corehost/fxr/fx_muxer.cpp

```diff
@@ -572,6 +572,11 @@ int fx_muxer_t::execute(
             return result;
         }
     }
+    else if (static_cast<StatusCode>(result) == AppArgNotManagedFailure)
+    {
+        trace::error(_X("The application '%s' does not exist or is not a managed .dll or .exe."), app_candidate.c_str());
```

Comment:
```suggestion
        trace::error(_X("The application '%s' is not a managed .dll or .exe."), app_candidate.c_str());
```
This error code is returned above only when the file actually exists